### ISSUE TITLE
[Scheduler] Add option to skip next occurence of recurring task.

### ISSUE THREAD

**Issue by jack1142** [mikeshardmind/SinbadCogs#137]

**Comment by mikeshardmind** [comment]
Could be accomplished relatively easily by modifying the task's original start date. Won't be included until after Red 3.1 release unless someone else PRs this before then.

**Comment by mikeshardmind** [comment]
Actually, on further though, I'd prefer adding a new attribute to the tasks for snoozing tasks. This would allow a more flexible use without killing the original task formatting. I'm gonna mark this blocked/related to the scheduler redesign planned instead.

**Comment by mikeshardmind** [comment]
Superseded by work to move this into core Red

**Comment by DiscordLiz** [comment]
Can I convince you to pick this back up when picking up the other scheduler issue I'm going to convince you to pick up?

**Comment by mikeshardmind** [comment]
> Actually, on further though, I'd prefer adding a new attribute to the tasks for snoozing tasks. This would allow a more flexible use without killing the original task formatting.

Blocked by #237 but needs to be considered during that
